### PR TITLE
[Link v1] Add ClickableBehavior to Link

### PR DESCRIPTION
**Pull request by chenyejia**
Move `ClickableBehavior` to `wonder-blocks-core` and add it to `Link`. Minor updates include fixing keyboard link behavior and updating focused state depending on last interaction.

Test Plan:
Tested Link behavior on styleguidist

### CODE REVIEW

**Comment by jeresig** [comment]
Deploy preview for *wonder-blocks* ready!

Built with commit 49f9cce10cd0043d3d3c8096020b026b022b8744

https://deploy-preview-86--wonder-blocks.netlify.com

File: packages/wonder-blocks-button/components/button-core.js [hunk 1]
```diff
@@ -5,8 +5,8 @@ import {StyleSheet} from "aphrodite";
 import {LabelLarge} from "wonder-blocks-typography";
 import Color, {mix, fade} from "wonder-blocks-color";
 import {addStyle} from "wonder-blocks-core";
+import type {Handlers} from "wonder-blocks-core";
```
Comment (jeresig) [inline packages/wonder-blocks-button/components/button-core.js]: Hmm - I feel like this name isn't descriptive enough. Maybe `ClickableHandlers` instead?

File: packages/wonder-blocks-core/components/clickable-behavior.js [hunk 2]
```diff
@@ -115,14 +115,13 @@ export default class ClickableBehavior extends React.Component<Props, State> {
     }
 
     handleClick = (e: SyntheticMouseEvent<>) => {
-        if (this.props.onClick) {
+        if (this.keyboardClick && e) {
```
Comment (jeresig) [inline packages/wonder-blocks-core/components/clickable-behavior.js]: Hmm - when would `e` be undefined? Shouldn't it always be defined? (especially as stated by the Flow rule)

Comment (chenyejia) [reply packages/wonder-blocks-core/components/clickable-behavior.js]: Double jk, it was actually because I wasn't passing in an object in the tests.

**Pull request merged**